src/azure-cli/azure/cli/command_modules/storage/_params.py

```diff
@@ -99,7 +106,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
     with self.argument_context('storage account check-name') as c:
         c.argument('name', options_list=['--name', '-n'])
 
-    with self.argument_context('storage account create') as c:
+    with self.argument_context('storage account create', resource_type=ResourceType.MGMT_STORAGE) as c:
```

Comment:
What will happen if resource_type is not specified

Reply:
It will not use the defined API version.